runtime/integration-tests/src/generic/envs/runtime_env.rs

```diff
@@ -92,7 +96,18 @@ impl<T: Runtime> Env<T> for RuntimeEnv<T> {
 			utils::create_extrinsic::<T>(who, call, nonce)
 		});
 
-		self.parachain_state_mut(|| T::Api::apply_extrinsic(extrinsic).unwrap())?;
+		self.parachain_state_mut(|| {
+			let res = T::Api::apply_extrinsic(extrinsic);
+			// NOTE: This is our custom error that we are having in the
+			//       `PreBalanceTransferExtension` SignedExtension, so we need to
+			//        catch that here.
+			if let Err(TransactionValidityError::Invalid(InvalidTransaction::Custom(255))) = res {
```

Comment:
Does `255` come from any constant defined somewhere else, or is it random?

Reply:
Random